src/main/java/com/jcabi/log/Logger.java

```diff
@@ -355,6 +357,24 @@ public void write(final int data) throws IOException {
     }
 
     /**
+     * Set final static field.
+     * @param field Field
+     * @param value New value
+     * @throws NoSuchFieldException If some problem
+     * @throws IllegalAccessException If some problem
+     */
+    // @checkstyle JavadocLocationCheck (2 lines)
```

Comment:
@pecko sorry, I misread checkstyle suppression name
but you done everything correctly